item-descriptions.lua

```diff
@@ -36,24 +36,23 @@ vanilla item in the game.
 
 If "raw/scripts/item-descriptions.lua" exists, it will entirely replace this one.
 Instead, mods should use "raw/scripts/more-item-descriptions.lua" to add content or replace
-descriptions on a case-by-case basis.  If an item description cannot be found in
+descriptions on a case-by-case basis. If an item description cannot be found in
 the latter script, view-item-info will fall back to the former.
 ]]
 
 -- see http://dwarffortresswiki.org/index.php/cv:Item_token
---luacheck: global
```

Comment:
Keep this in. It's necessary for df-luacheck stuff that @BenLubar is working on, apparently.

Reply:
Good catch. Apparently I based the edit off of some old version of the file, and this got lost in transition.

Reply:
Yeah, I figured that was the most likely cause.